virttest/openvswitch.py

```diff
@@ -473,30 +476,65 @@ def init_db(self):
         process.run('%s %s %s %s %s' %
                     (path.find_command("ovsdb-server"),
                      "--remote=punix:%s" % (self.db_socket),
-                     "--remote=db:Open_vSwitch,manager_options",
+                     "--remote=db:Open_vSwitch,Open_vSwitch,manager_options",
                      "--pidfile=%s" % (self.db_pidfile),
-                     "--detach"))
+                     "--detach --log-file %s") % (self.db_path))
         self.ovs_vsctl(["--no-wait", "init"])
 
     def start_ovs_vswitchd(self):
+        """
+        Start ovs vswitch daemon with only 1 socket
+        """
+        self.ovs_vsctl(["--no-wait", "set", "Open_vSwitch",
+                        ".", "other_config:dpdk-init=true"])
+        self.ovs_vsctl(["--no-wait", "set", "Open_vSwitch",
+                        ".", "other_config:dpdk-socket-mem='1024'"])
+        self.ovs_vsctl(["--no-wait", "set", "Open_vSwitch",
+                        ".", "other_config:dpdk-lcore-mask='0x1'"])
         process.run('%s %s %s %s' %
                     (path.find_command("ovs-vswitchd"),
                      "--detach",
                      "--pidfile=%s" % self.ovs_pidfile,
                      "unix:%s" % self.db_socket))
 
+    def create_br_port(self, br_name, port_names):
```

Comment:
suggest separate creating and port creating into two functions. In this way, each time I want to add port, it need create bridge again.

Reply:
updated, thanks